docs/zed/api.md

```diff
@@ -200,42 +462,26 @@ data: {"pool_id": "1sMDXpVwqxm36Rc2vfrmgizc3jz", "commit_id": "1tisISpHoWI7MAZdF
 
 event: pool-delete
 data: {"pool_id": "1sMDXpVwqxm36Rc2vfrmgizc3jz"}
-
-```
-
-### Index Rules
-
-#### Create Index Rule
-
-Creates an index rule for a specified field.
-
 ```
-POST /index
-```
-
-#### Delete Index Rule
 
-Deletes the specified index rule. Any created object indexes will persist.
-
-```
-DELETE /index
-```
```

Comment:
I reviewed this one with @mattnibs before proposing to remove it in this PR. What I observed via black box testing was that the `zed index create` seemed to be cooking up its KSUIDs and timestamps client-side and posting them along with the new rule details. I'm not a pro at interpreting the code, but I think these snippets confirm that:

[https://github.com/brimdata/zed/blob/7e9f5e5a51ee061e7362244a5245bbdd9766a3a3/cmd/zed/index/create.go#L109](https://github.com/brimdata/zed/blob/7e9f5e5a51ee061e7362244a5245bbdd9766a3a3/cmd/zed/index/create.go#L109)  
[https://github.com/brimdata/zed/blob/7e9f5e5a51ee061e7362244a5245bbdd9766a3a3/lake/index/rule.go#L50-L55](https://github.com/brimdata/zed/blob/7e9f5e5a51ee061e7362244a5245bbdd9766a3a3/lake/index/rule.go#L50-L55)  
[https://github.com/brimdata/zed/blob/7e9f5e5a51ee061e7362244a5245bbdd9766a3a3/lake/index/rule.go#L23-L27](https://github.com/brimdata/zed/blob/7e9f5e5a51ee061e7362244a5245bbdd9766a3a3/lake/index/rule.go#L23-L27)  
  
If I'm right about that, this seems like it would be awkward to explain to an API user and ask them to adapt to, so we agreed that for now we could just leave these endpoints undocumented until @mattnibs has some time to look closer, and maybe the API will change regardless.

Reply:
Hadn't had a chance to look at this in depth, but if we are cooking up ksuids client side this should be changed.